fa20/syllabus.yml

```diff
@@ -2,17 +2,18 @@
   required:
     id: 8f863b4f2ad5d2600cb549cf885be1065f9423efbd7526a3f3e73457fab26bb7
     date: !Timestamp 2020-09-15 18:00:00
-    title: meeting00
+    title: a-few-useful-things
     authors: ["PaulLunis]
-    filename: meeting00
+    filename: useful-ml
     cover-image: 'https://cacm.acm.org/system/assets/0000/9220/092012_CACMpg79_A-Few-Useful-Things2.large.jpg?1476779416&1348149730'
-    tags: ["ai","machine learning"]
+    tags: ["ai","machine learning","review paper","data science","ml research"]
     room: Online
     abstract: >-
      This week for our first paper we'll be reading A Few Useful Things to Know about Machine Learning. This will be a useful review paper to help get 
      everyone familiar with the field of machine learning and is full of useful vocabulary and conventional outlooks on AI as a whole. 
   optional:  # All `optional` keys are enumerated in the Documentation
-    papers: https://homes.cs.washington.edu/~pedrod/papers/cacm12.pdf
+    papers:
+      useful ml pdf: https://homes.cs.washington.edu/~pedrod/papers/cacm12.pdf
```

Comment:
I'll have to get back to you on whether this bricks things. I've always defaulted to either `camelCase` or `putting-dashes-in-names` 'cause this becomes an on-disk file.